### ISSUE TITLE
ProcServ: reproduce issue with "bad" permissions

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#8555]
# Issue Description

As a developer I would like procserv log files to be reliably created with correct permissions.

On many instruments, particularly (but not exclusively) at the time of starting the IBEX server, the procserv log files can be created with invalid permissions. The symptoms are that:
- Right click -> properties -> permissions is unable to show file permissions correctly
- `dir /q` reports `...` as the file owner rather than an actual username
- The files are not readable by anyone (including admin)

Although this is happening on most instruments, it is _not_ happening on dev or test machines. This ticket is to find a _reliable_ way to reproduce the problem, ideally without an instrument machine, so that we can investigate/fix it.

Although this was noticed in the context of log rotation, it's actually a more general problem as it would prevent us from looking at these log files to diagnose a problem.

### How & Where?

procServ is in `EPICS-tools`.

### Reproducible (Yes/No)?

No: finding a way to reproduce this reliably is part of this issue.

# Acceptance Criteria
- [ ] Documented reliable way to trigger this issue

Either:
- [ ] Issue fixed
or
- [ ] Follow-up issue created & proposed to fix

# How to Review

Check that you can reliably reproduce this issue.

**Comment by FreddieAkeroyd** [comment]
From looking on an instrument it seems odd permission files are being held open. Running handle indicated that some of these log files were locked by the ioc and cmd.exe not procserv, theory it is they inherit the old log file handle during a fork and then keep it after the parent rotates to a new day

**Comment by FreddieAkeroyd** [comment]
Yes, to be sure we should park this ticket for now and add a task to check on the log rotation behaviour and on disk log files of some of the instruments that are deployed to next cycle. If all is then looking good on those instruments we should then close the ticket as no longer needed.

**Comment by iangillingham-stfc** [comment]
Task entered to check at some future date
https://github.com/orgs/ISISComputingGroup/projects/17/views/3?pane=issue&itemId=91475032
